src/Pokemon.java

```diff
@@ -0,0 +1,68 @@
+public abstract class Pokemon {
```

Comment:
Goed dat je het abstract hebt gemaakt. Je had ook nog abstracte methodes kunnen maken, zoals `getAttacks` of helper methodes